### PR TITLE
normative statements: Core properties section

### PR DESCRIPTION
**Pull request by rhiaro**
Normative statement tweaks in the core properties section based on feedback from @kdenhartog and @msporny; see #384

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/429.html" title="Last updated on Oct 18, 2020, 3:57 PM UTC (7cf35fc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/429/c764c67...7cf35fc.html" title="Last updated on Oct 18, 2020, 3:57 PM UTC (7cf35fc)">Diff</a>

### CODE REVIEW

File: index.html
Comment (msporny) [inline index.html:1777]: Hmm, why are we removing this? It's testable via a test that attempts to use a verification method listed in `verificationMethods` to perform a verification where the verification relationship does not appear in the DID Document. For example, using a capability invocation key for the purposes of authentication.

**Review by msporny — CHANGES_REQUESTED** [review]
Overall, agree with the changes save for one.

**Comment by msporny** [comment]
@rhiaro waiting for a response on https://github.com/w3c/did-core/pull/429#discussion_r499181729 before proceeding.

File: index.html
Comment (kdenhartog) [reply index.html:1777]: Looking at the comment I left on the sheet, it looks like the reason we'd remove it is because this is a statement testing software that relies upon the did document rather than a did document processor. E.g. a statement that would describe how to write a VC verification library.

Doing this seemed odd to me so I left that comment. If you feel it would be beneficial to leave it as normative though, I'm alright with that.

**Review by kdenhartog — APPROVED** [review]
Happy with the changes. Willing to approve if we keep the MUST NOT statement as well.

File: index.html
Comment (rhiaro) [reply index.html:1777]: @kdenhartog's feedback in the spreadsheet on this line was "Make the wording non-normative but leave in there" so I did that. Sounds like Kyle has changed his mind though? I'll revert the commit.

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
